docs/monitoring/reports-and-components.md

```diff
@@ -22,7 +22,7 @@ Split screen gives you a single view for Metrics, Logs, and Events. This is a hu
 
 ![Sematext Cloud Monitoring Guide - Monitoring Split Screen](../images/guide/monitoring/monitoring-split-screen.png) 
 
-Combined with the **Metric Reports**, and with the help of a **charting hairline**, zoom and chart navigation tools, it is easy to see how using our **Monitoring and Logging tools in a single page** is a must-have for any DevOps team! Learn more about **[split screen](./split-screen/)**.
+Combined with the **Metric Reports**, and with the help of a **charting hairline**, zoom and chart navigation tools, it is easy to see how using our **Monitoring and Logging tools in a single page** is a must-have for any DevOps team! Learn more about **[split screen](/guide/split-screen)**.
```

Comment:
How come these two links are different? One is `#split-screen` while other is `/guide/split-screen`

Reply:
because I thought they would have different purposes. The first one is on the "intro" section, so I guess the user would want to jump to the heading (`#split-screen`) without being thrown off-context. The second one is "learn more" so I guess the user would want to jump to the separate (more detailed) page. He is already in the section.

Reply:
Ack!